src/utils/viewer/viewer.cpp

```diff
@@ -156,6 +169,7 @@ Key Commands:
   'q': Query the agent's state and print to terminal.
 
   Utilities:
+  'num5' switch ortho/perspective camera.
```

Comment:
Is this numpad 5? What about people on laptops?

Reply:
Just using what mosra suggested above.  No skin off my nose either way.

Reply:
I think the reasoning was that it is the same as what blender uses.

Reply:
I would pick another one.

Reply:
I originally had 'c' :)

Reply:
In blender there is a function called "Emulate numpad". We do not have it here.
How about just using number '5'? KeyEvent::Key::Five

Reply:
> What about people on laptops?

I have a numpad on my laptop, what's the problem? :trollface: 

Numpad five was chosen because other numpad keys switch between front/top/side views. Five alone doesn't have any meaning, and it would have *actual* issues on my layout because I'd need to do <kbd>Shift</kbd> + <kbd>ř</kbd> to get 5.

I'd say go back to what you had before, then.